src/types/index.ts

```diff
@@ -141,6 +141,7 @@ export interface ITokenBalanceResponse {
 
 /**
  * Fungible tokens that are available in currently used network.
+ * Including the native coin.
```

Comment:
This is actually not true... kind of. Because we have the `ICoin` which initially was describing the coins. But almost since I joined the team I really didn't like this as the `ICoin` is the `IToken` with coin gecko data :/

 I'm in an constant process of replacing the `ICoin` and `IToken` with only one type `IAsset` which for now is equal to `ICoin | IToken`. In the future there will be only `IAsset`, and when this happens the line you added will make sense. For now it actually messes the concept of the difference between `ICoin` and `IToken`.